spec/project_spec.rb

```diff
@@ -233,13 +233,23 @@ def expected
   describe 'Changelog' do
     subject(:changelog) { File.read(path) }
 
-    let(:path) { File.join(File.dirname(__FILE__), '..', 'CHANGELOG.md') }
+    let(:path) { File.expand_path('../CHANGELOG.md', __dir__) }
     let(:entries) { lines.grep(/^\*/).map(&:chomp) }
 
     include_examples 'has Changelog format'
 
     context 'future entries' do
-      dir = File.join(File.dirname(__FILE__), '..', 'changelog')
+      let(:all_cop_names) do
```

Comment:
We cannot use top-level `cop_names` because it excludes `InternalAffairs` which is interesting for us.